soc/arm64/intel_socfpga/common/soc.h

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
-1 on adding empty files, this is likely papering over a real issue

Reply:
> -1 on adding empty files, this is likely papering over a real issue

Was trying to test if adding the soc.h would remove the CI error, not intended to add the soc.h back into the repo.